SteamKit2/SteamKit2/Steam/Handlers/SteamApps/Callbacks.cs

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
can you undo these changes please (utf BOM probably)

Reply:
Fixed that